internal/cloudapi/v2/errors.go

```diff
@@ -123,6 +124,7 @@ func getServiceErrors() serviceErrors {
 		serviceError{ErrorUnsupportedImage, http.StatusBadRequest, "This compose doesn't support the creation of multiple images"},
 		serviceError{ErrorInvalidImageFromComposeId, http.StatusBadRequest, "Invalid format for image id"},
 		serviceError{ErrorImageNotFound, http.StatusBadRequest, "Image with given id not found"},
+		serviceError{ErrorUnreachableImage, http.StatusBadRequest, "You need to use either share_with_accounts, or public"},
```

Comment:
The message should be probably `"You need to specify at least one of share_with_accounts, or public"`. IOW the expectation is not to specify one or the other exclusively, but at least one of them...